extensions/wikia/ARecoveryEngine/js/SourcePoint/slotFinder.js

```diff
@@ -1,5 +1,5 @@
 /*global define*/
-define('ext.wikia.aRecoveryEngine.recovery.slotFinder', [
+define('ext.wikia.aRecoveryEngine.sourcePoint.slotFinder', [
```

Comment:
Right, I alarmed "automatically" because I saw the rename. Sorry about that and thanks! :)